Storage/SQL/SQLiteLogins.swift

```diff
@@ -249,6 +249,37 @@ public class SQLiteLogins: BrowserLogins {
         return db.runQuery(sql, args: args, factory: SQLiteLogins.LoginDataFactory)
     }
 
+    public func getAllLogins() -> Deferred<Maybe<Cursor<LoginData>>> {
+        return searchLoginsByUsername(nil)
+    }
+
+    public func searchLoginsByUsername(username: String?, orPassword password: String? = nil, orHostname hostname: String? = nil) -> Deferred<Maybe<Cursor<LoginData>>> {
```

Comment:
Actually now that I think of it, probably not since we're just doing an OR. I can use the provided string across the three columns instead.